components/sections/derivatives/content.en-GB.md

```diff
@@ -73,12 +73,12 @@ And that's the first part done: the two components inside the parentheses are ac
 Now to apply this to our weighted Bezier curves. We'll write out the plain curve formula that we saw earlier, and then work our way through to its derivative:
 
 \[\begin{array}{lcl}
-  Bézier_{n,k}(t) &=& B_{n,0}(t) \cdot w_0 + B_{n,1}(t) \cdot w_1 + B_{n,2}(t) \cdot w_2 + B_{n,3}(t) \cdot w_3 + ... \\
-  Bézier_{n,k}(t) \frac{d}{dt} &=& n \cdot (B_{n-1,-1}(t) - B_{n-1,0}(t)) \cdot w_0 + \\
-                               & & n \cdot (B_{n-1,0}(t) - B_{n-1,1}(t)) \cdot w_1 + \\
-                               & & n \cdot (B_{n-1,1}(t) - B_{n-1,2}(t)) \cdot w_2 + \\
-                               & & n \cdot (B_{n-1,2}(t) - B_{n-1,3}(t)) \cdot w_3 + \\
-                               & & ...
+  Bézier(n,t)  &=& B_{n,0}(t) \cdot w_0 + B_{n,1}(t) \cdot w_1 + B_{n,2}(t) \cdot w_2 + B_{n,3}(t) \cdot w_3 + ... \\
```

Comment:
the corrections here need to be reverted, as the expansions all rely on `_{n,k}(t)` notation. We can't just remove that from the left hand side.

Reply:
Hmm, how about writing the last term explicitly?

<img width="744" alt="bezier" src="https://user-images.githubusercontent.com/6268183/29995973-911156a8-9030-11e7-9aaf-8042145203ca.png">

Reply:
That would work, although your example seems to have lost the `n-1` aspect, instead using `n`.